data/resources/style.css

```diff
@@ -263,14 +269,21 @@ messagemedia overlay > messageindicators, messagesticker messageindicators {
 
 /* Placeholder */
 .message-entry > overlay > label {
-  margin: 0 9px;
+  margin-left: 9px;
+}
+
+.message-entry > overlay > entry > image {
+  margin: 9px;
```

Comment:
The left margin should not be there, otherwise the scrollbar is placed too far away:

![image](https://user-images.githubusercontent.com/23530586/195696181-c09295f8-5b38-4aac-9f0c-34b41abcf3d0.png)